packages/namekit-react/src/components/TruncatedText.tsx

```diff
@@ -62,10 +62,10 @@ export const TruncatedText = ({
 
   const getTextElm = (
     classes: string,
-    maxWidth = maxDisplayWidth,
+    maxWidth?: number,
```

Comment:
The strategy used to fix the bug I reported previously wasn't correct. But I fixed it here.

This is the proper way to ensure that the `maxDisplayWidth` is never used as a default `maxTooltipWidth`.